src/MultiTypeSymbolTable.chpl

```diff
@@ -20,7 +20,7 @@ module MultiTypeSymbolTable
 
     private config const logLevel = ServerConfig.logLevel;
     private config const logChannel = ServerConfig.logChannel;
-    const mtLogger = new Logger(logLevel, logChannel);
+    const mtLogger : owned Logger = new Logger(logLevel, logChannel);
```

Comment:
IIRC, this was an earlier attempt to solve the issue. If that's the case, I have a mild preference for reverting this.